protocols/direct-fund/direct-fund_test.go

```diff
@@ -216,8 +216,8 @@ func TestCrank(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	if waitingFor != WaitingForMyTurnToFund {
-		t.Errorf(`WaitingFor: expected %v, got %v`, WaitingForMyTurnToFund, waitingFor)
+	if waitingFor != WaitingForCompleteFunding {
+		t.Errorf(`WaitingFor: expected %v, got %v`, WaitingForCompleteFunding, waitingFor)
```

Comment:
Does this change belong on this PR?

Reply:
This has been pulled out into https://github.com/statechannels/go-nitro/pull/215